sycl/include/CL/sycl/detail/cg_types.hpp

```diff
@@ -246,6 +246,7 @@ template <class KernelType, class KernelArgType, int Dims>
 class HostKernel : public HostKernelBase {
   using IDBuilder = sycl::detail::Builder;
   KernelType MKernel;
+
```

Comment:
here and in other places - please avoid unrelated changes like adding empty lines